docs/troubleshooting/troubleshooting.md

```diff
@@ -1,4 +1,69 @@
+    mongodb-67bf769957-9ct5z       1/1     Running     0               8m10s
+    scheduler-654cbd4b97-qwtbf     1/1     Running     0               8m10s
+    terraformer-fd664b7ff-dd2h7    1/1     Running     0               8m9s
+    ```
+
+1. After verifying if any pods in a failed state, you can check the logs of the frontend service for each service. Typically, if all Claudie pods are scheduled but cluster creation fails, it may be due to missing permissions on the credentials used for Claudie providers.
+
+    ```bash
+    kubectl -n claudie logs -l app.kubernetes.io/name=frontend
+    ```
+
+    ```text
+    6:04AM INF Using log with the level "info" module=frontend
+    6:04AM INF Frontend is ready to process input manifests module=frontend
+    6:04AM INF Frontend is ready to watch input manifest statuses module=frontend
+    ...
+    9:19AM WRN Retrying command terraform apply --auto-approve... (5/10) module=terraformer
+    9:20AM WRN Error encountered while executing terraform apply --auto-approve : exit status 1 module=terraformer
+    9:20AM INF Next retry in 300s... module=terraformer
+    9:25AM WRN Retrying command terraform apply --auto-approve... (6/10) module=terraformer
+    ```
+
+    !!! note "Debug log level"
+        Using debug log level will help here with identifying the issue closely. [This guide](https://docs.claudie.io/v0.4.0/getting-started/detailed-guide/#claudie-deployment) shows how you can set it up during step 5.
+
+    !!! node "Claudie benefit!"
+        The great thing about Claudie is that it utilizes open source tools to set up and configure infrastructure based on your preferences. As a result, the majority of errors can be easily found and resolved through online resources.
+
+### Terraformer service not starting
+Terraformer relies on minio and dynamodb services for proper startup. If these services fail to start, Terraformer will also fail to start.
+
+### Create table job
+Scheduling problems of dynamodb service or even very slow autoscaling, may cause `create-table-job` to fail to create necessary tables in dynamodb service in time. We have set the `backoffLimit` of the `create-table-job` to fail after approximately 42 minutes of trying. If you encounter any issues with the `create-table-job` or if you feel like the `backoffLimit` has to be extended, please [create an issue](https://github.com/berops/claudie/issues).
+
+## Networking issues
+### Wireguard MTU
+We use Wireguard for secure node-to-node connectivity. However, it requires setting the MTU value to match that of Wireguard. While the host system interface MTU value is adjusted accordingly, networking issues may arise for services hosted on Claudie managed Kubernetes clusters. For example, we observed that the GitHub actions runner docker container had to be configured with an MTU value of `1380` to avoid network errors during `docker build` process.
+
+### Hetzner and OCI node pools
```

Comment:
This affects mainly Ansibler and Kube-eleven. The Kuber is actually fine even with bad IPs.